rllib/utils/test_utils.py

```diff
@@ -718,9 +723,13 @@ def run_learning_tests_from_yaml(
                 ])
 
                 throughput = timesteps_total / (total_time_s or 1.0)
-                desired_throughput = None
-                # TODO(Jun): Stop checking throughput for now.
-                # desired_throughput = checks[experiment]["min_throughput"]
+                desired_throughput = checks[experiment]["min_throughput"]
```

Comment:
Maybe for a later PR: Should we measure env-throughput and learner-throughput separately?